src/redis/local_cache_stats.go

```diff
@@ -0,0 +1,43 @@
+package redis
```

Comment:
This file needs actually tests like we do for other stats. Please add coverage (integration or otherwise) that actually make use of the stats and verify them.

Reply:
Just embedded the test into the existing test case `TestOverLimitWithLocalCache`,  and we can better compare new-added local_cache stats with the existing stats.